api/capi/src/nnstreamer-capi-single.c

```diff
@@ -268,8 +271,11 @@ ml_single_open (ml_single_h * single, const char *model,
   single_h->sink = appsink;
   single_h->filter = filter;
   single_h->timeout = SINGLE_DEFAULT_TIMEOUT;
+  single_h->clear_previous_buffer = FALSE;
+  single_h->is_valid = TRUE;
```

Comment:
This should be set TRUE only when everything is safe; at the last step right before "return OK".